src/main/java/org/Quotes/Main.java

```diff
@@ -0,0 +1,113 @@
+package org.Quotes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Main Application
+ */
+public class Main {
+    /**
+     * The path of the file to store the quote of the day
+     */
+    private static final String FILE_PATH = Paths.get("Data", "quote_of_the_day.json").toString();
+    /**
+     * Main class for accessing the Programming Quotes API for quote of the day
+     *
+     * @param args not used
+     */
+    public static void main(String[] args) {
+        try {
+            // Check if the file exists and is not empty
+            File file = new File(FILE_PATH);
+            if (file.exists() && file.length() > 0) {
+                // Read the quote of the day from the file
+                String json = new String(java.nio.file.Files.readAllBytes(Paths.get(FILE_PATH)));
+                Gson gson = new Gson();
+                Quote quote = gson.fromJson(json, Quote.class);
+                String fileDate = quote.contents.quotes[0].date;
+                String todayDate = LocalDate.now().format(DateTimeFormatter.ISO_DATE);
+                if (fileDate.equals(todayDate)) {
+                    System.out.println("From The .json File:");
+                    System.out.println("Today's (" + quote.contents.quotes[0].date + ") Quote:");
+                    System.out.println('"'+quote.contents.quotes[0].quote+'"');
+                    System.out.println("By : " + quote.contents.quotes[0].author);
+                }
+                else {
+                    System.out.println("Found: \n(" + quote.contents.quotes[0].date + ") Quote: ");
+                    System.out.print('"'+quote.contents.quotes[0].quote+'"');
+
+                    System.out.println("Updating the quote !!");
+                    // Delete the file
+                    file.delete();
```

Comment:
Maybe we can get away with overwriting the file

Reply:
👍🏻